src/heretic/r_bsp.c

```diff
@@ -313,7 +313,7 @@ void R_AddLine(seg_t * line)
 // reject empty lines used for triggers and special events
     if (backsector->ceilingpic == frontsector->ceilingpic
         && backsector->floorpic == frontsector->floorpic
-        && backsector->lightlevel == frontsector->lightlevel
+        && backsector->rlightlevel == frontsector->rlightlevel
```

Comment:
Required to avoid flickering glitch in E1M1 Hallway where Gauntlets are located. Should be fixed in crispy-doom aswell?

Reply:
It's probably not even an oversight, as I wanted to reduce changes to apparently unrelated code paths to a minimum. My initial idea was to let the rest of the game act on the unmodified light levels, but if this leads to glitches, these should of course get fixed. Have you found similar glitches in Doom already?

Reply:
>  Have you found similar glitches in Doom already?

Yes, e. g. in Doom2 Level 09 near the yellow key-card:
![2024-12-15 12_45_01-Window](https://github.com/user-attachments/assets/3484d823-69fb-420d-a949-17ae3c58de46)

Reply:
Oh, wow! Yes, this needs to get fixed, please!